File: src/main/java/um/haberes/report/model/haberes/core/ContratadoPersonaDto.java
package um.haberes.report.model.haberes.core;

import java.math.BigDecimal;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ContratadoPersonaDto {

    private Long contratadoId;

    private BigDecimal personaId;

    private Integer documentoId;

    private String apellido;

    private String nombre;

    private String cuit;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/ContratadoPersonaDto.kt
package um.haberes.report.kotlin.dto.haberes.core

import java.math.BigDecimal

data class ContratadoPersonaDto(

    var contratadoId: Long? = null,
    var personaId: BigDecimal? = null,
    var documentoId: Int? = null,
    var apellido: String? = null,
    var nombre: String? = null,
    var cuit: String? = null

)
